website_sale_invoice_address/views/templates.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="website_sale_invoice_address.confirmation"
+        inherit_id="website_sale.confirmation"
+        priority="100"
+    >
+
+        <xpath expr="//span[hasclass('address-inline')]" position="replace">
```

Comment:
Why not simply change the attribute itself?
```suggestion
        <xpath expr="//span[hasclass('address-inline')]" position="attributes">
```

Reply:
I've made the corrections, thanks for your review!